### PR TITLE
feat(store): add transformer from v2 config output to v1 config output

### PR DESCRIPTION
**Pull request by alvrs**

### CODE REVIEW

File: test/mock-game-contracts/mud.config.ts [hunk 1]
```diff
@@ -114,3 +114,5 @@ export const configV2 = resolveStoreConfig({
     },
   },
 });
+
+export default config;
```
Comment (yonadaaa) [inline test/mock-game-contracts/mud.config.ts:118]: OOC what's the reason for this change? just a style thing?

Comment (alvrs) [reply test/mock-game-contracts/mud.config.ts:118]: oh this was just a temporary thing to be able to inspect individual fields of the config output better
